book/hardware-batterybay.md

```diff
@@ -1,8 +1,8 @@
-# Battery Bay Interface
+# Battery Bay
 
-## Overview
+The *Battery Bay* is the intended location for mounting power system accessories including high capacity batteries, tethers, fuel cells and alternative energy sources. 
 
-The battery bay is the ideal location for mounting power system accessories like high capacity batteries, tethers and fuel cells or alternative energy sources. Since power system payloads are useful for all applications it is very important to make sure it does not interfere with the keep out zones of the [Accessory Bay](http://dev.3dr.com/hardware-accessorybay.html).
+Since power system payloads are useful for all applications it is very important to make sure it does not interfere with the keep out zones of the [Accessory Bay](hardware-accessorybay.md).
```

Comment:
Is this really necessary to say? 
* The accessory bay is below the vehicle. If you're going to say this why not also say avoid the gimbal bay?
* What we should mention though is the propeller area, since that is at risk?
* Or is this a generic statement because theoretically someone might wrap their battery around the vehicle?

If we do want to say something along these lines, I'd propose we have a generic comment about all the other bays/areas in the section "Mechanical Interface Specifications"